syncr_frontend/frontend.py

```diff
@@ -2,13 +2,13 @@
 import subprocess
 from os import path
 from os import scandir
-from tkinter import filedialog
-from tkinter import Tk
 
 from flask import flash
 from flask import Flask
 from flask import render_template
 from flask import request
+from tkinter import filedialog
```

Comment:
is this still used?

Reply:
it's used in select_directory...not sure if tryjankowski was using that for his create drop / subscribe to drop stuff.

If not, I could totally just delete select_directory and the TK stuff

Reply:
I'll text him and see if he uses it